Libplanet.Common/EnumerableExtensions.cs

```diff
@@ -1,4 +1,3 @@
-#nullable disable
```

Comment:
I think we can get rid of `EnumerableExtensions` entirely. It's only used in one place, and it's probably better rephrase the LINQ expression in `Swarm`. 🙄